### DISCUSSION TITLE
ListItem.Subtitle theming?

### DISCUSSION THREAD

**Discussion by tamis-laan** [react-native-elements/react-native-elements#3605] [Q&A] [answered]
I'm trying to apply theming to react native elements. I have gone for a global dark theme:
``` javascript
{
		primary: 'grey',
		secondary: 'grey',
		white: 'black',
		black: 'white',
		grey0: '#ccc',
		grey1: '#bbb',
		grey2: '#aaa',
		grey3: '#999',
		grey4: '#333',
		grey5: '#111',
		greyOutline: '#333',
		searchBg: '#111',
		success: 'green',
		error: 'red',
		warning: 'yellow',
		divider: 'grey',
	}
```

However the color of `ListItem.subtitle` remains black so I'm trying to turn it white:
``` javascript
	ListItem: {
		Subtitle: {
			color: 'white'
		}
	}
```

It doesn't work, anyone know how to do this?

**Comment by arpitBhalla** [comment, marked as answer]
Use 
```
ListItemSubtitle:{
...
}
```

EDIT: added themeKey for each component in docs

**Reply by tamis-laan** [reply]
aaaaaah where is this in the docs? And a question is why doesn't the color object have an effect on subtitle?

**Reply by Nantris** [reply]
Definitely needs to be squeezed into the docs somewhere. Maybe not the _best_ place, but a single place it could be discussed: https://reactnativeelements.com/docs/customization/theme%20object

Or maybe at the bottom of `ThemeProvider`'s page since it will probably be more viewed: https://reactnativeelements.com/docs/customization/themprovider

Btw @arpitBhalla the URL there is mispelled, `/themprovider` should be `/themeprovider`
